Libraries/NativeAnimation/RCTNativeAnimatedNodesManager.m

```diff
@@ -264,6 +265,20 @@ - (void)stopAnimation:(nonnull NSNumber *)animationId
   }
 }
 
+- (void)stopAnimationsForNode:(nonnull RCTAnimatedNode *)node
+{
```

Comment:
Probably won’t make a big difference because active animations count is usually low but we should use a normal for loop like you did on android to avoid the extra array and removeObject (o(n)) calls.